README.md

```diff
@@ -29,6 +29,9 @@ Starting with Ansible 2.7, Ansible only supports Python 2.7 and 3.5 (and higher)
 
 ### Known issues
 
+* Some modules, mainly `katello_sync`, trigger long running tasks on the server side. It might be beneficial to your playbook to wait for their completion in an asynchronous manner.
```

Comment:
sync and cv_version, should both be mentions?

Reply:
Like this?
```suggestion
* Some modules, e.g. `katello_sync` and `katello_content_view_version`, trigger long running tasks on the server side. It might be beneficial to your playbook to wait for their completion in an asynchronous manner.
```

Reply:
Yes